dissect/target/helpers/utils.py

```diff
@@ -162,3 +162,9 @@ def year_rollover_helper(
                 continue
 
             yield relative_ts.replace(tzinfo=tzinfo), line
+
+
+def sanitize_string(value: str) -> str:
```

Comment:
Maybe something along the lines of `escape_null()` might be more striking? Since it only does that at the moment.